CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # main
 
+- New function `replicate!` allows to create a new instance of a given agent at the same position with the possibility to specify some
```

Comment:
Should have been a `# v5.17` here, you can push it to master directly

Reply:
ops sure :-)